### PR TITLE
Updated README and folder structure

### PR DESCRIPTION
**Pull request by bbaltz505**

### CODE REVIEW

**Comment by bbaltz505** [comment]
@eriknyquist @calvinatintel pls review the README and try out the structure

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Curie Open Development Kit - Z
+
+### Contents
+
+  - Firmware: Zephyr application ()
```
Comment (calvinatintel) [inline README.md]: ()?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ - Ubuntu 14.04 - 64 bit
+
+### Installation
+```
+mkdir CODK && cd $_
+git clone https://github.com/01org/CODK-Z.git
+cd CODK-Z
+sudo make install-dep
+make setup
+source ../zephyr/zephyr-env.sh
```
Comment (eriknyquist) [inline README.md]: Is there anything stopping us from hiding this in the Makefile?

**Comment by calvinatintel** [comment]
Merging in to work on separating out the utils into a standalone flashpack repo

**Comment by calvinatintel** [comment]
Overridden with https://github.com/01org/CODK-Z/pull/9

**Pull request closed without merging**